### PR TITLE
Add buttons for copying canonical package URIs

### PR DESCRIPTION
**Pull request by nl0**
copy current revision's uri:
Screenshot 2020-12-17 at 12 54 01

copy any revision's uri:
Screenshot 2020-12-17 at 12 54 23

# TODO

- [ ] [Changelog](CHANGELOG.md) entry
- [ ] wordsmith @akarve

### CODE REVIEW

**Comment by nl0** [comment]
@akarve pls wordsmith

File: catalog/app/containers/Bucket/PackageTree.js [hunk 1]
```diff
@@ -84,6 +88,15 @@ function RevisionInfo({ revisionData, revision, bucket, name, path }) {
     _: R.identity,
   })
 
+  const getHttpsUri = (r) =>
+    `${window.origin}${urls.bucketPackageTree(bucket, name, r.hash, path)}`
```
Comment (fiskus) [inline catalog/app/containers/Bucket/PackageTree.js:92]: Hm... I expected `quilt+s3://` url here

Comment (fiskus) [reply catalog/app/containers/Bucket/PackageTree.js:92]: Now it's the same URL as if you copy it by right click and select "Copy link location"

Screenshot from 2020-12-17 11-34-40

**Review by fiskus — COMMENTED** [review]
Code is fine. But I expected different functionality: copying immutable URI

**Review by fiskus — APPROVED** [review]
Found the discussion in Slack

**Pull request merged**